AUTHORS

```diff
@@ -3361,5 +3361,13 @@ Francesco Trotta <github@fasttime.org>
 Richie Bendall <richiebendall@gmail.com>
 ryan <chai_xb@163.com>
 Piotr Rybak <rybak.piotr@yahoo.com>
+Jean Burellier <sheplu@users.noreply.github.com>
+Subhi Al Hasan <subhialhasan@protonmail.com>
+3nprob <3nprob@3nprob>
+Martin Jansa <martin.jansa@lge.com>
+Omar El-Mihilmy <mihilmy@yahoo.com>
+Simone Busoli <simone.busoli@gmail.com>
```

Comment:
This one is a duplicate so we need to add a .mailmap entry instead. @simoneb Do you have a preference as to whether we keep your gmail.com address or your nearform.com address?

Reply:
Iam have some issues running installing with out errors

Reply:
> Iam have some issues running installing with out errors

I have blocked this bot from the Node.js org.